plugin/src/main/java/io/snyk/languageserver/protocolextension/messageObjects/scanResults/AdditionalData.java

```diff
@@ -0,0 +1,149 @@
+package io.snyk.languageserver.protocolextension.messageObjects.scanResults;
+
+public class AdditionalData {
```

Comment:
I think these classes need `equals` and `hashCode` methods to be able to sort & compare them